.github/workflows/lighthouse.yml

```
@@ -0,0 +1,20 @@
+name: Lighthouse CI
+on: [push]
+jobs:
+  lhci:
+    name: Lighthouse
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v1
```

```suggestion
        uses: actions/setup-node@v3
```
v1 is very outdated.